src/test/java/com/google/sps/data/InstanceTest.java

```diff
@@ -56,7 +58,7 @@ public void testConversionBetweenEntityAndInstance() {
     InstanceInterface instance = new Instance("vm1", "Running", sessionId);
     Entity instanceEntity = instance.toEntity();
     Instance newInstance = Instance.fromEntity(instanceEntity);
-    Assert.assertTrue(instance.isEqualTo(newInstance));  
+    Assert.assertTrue(instance.equals(newInstance));  
```

Comment:
You should be able to do:

Assert.asserEquals(instance, newInstance);

That should implicitly call equals() for you